graphql/project_resolver.go

```diff
@@ -46,6 +46,8 @@ func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProject
 		if err != nil {
 			return nil, InternalServerError.Send(ctx, fmt.Sprintf("problem building APIPatch from service for patch: %s : %s", p.Id.Hex(), err.Error()))
 		}
+		// Only show the enqueue button if the project uses the Evergreen commit queue.
```

Comment:
Can you put this logic into buildFromService?
https://github.com/evergreen-ci/evergreen/blob/0e4178aa0c4bd3b7690a351fca870efcb31bf41a/rest/model/patch.go#L275

Reply:
done